### PR TITLE
Init Sprint 1

### PR DESCRIPTION
**Pull request by EvanMateo7**
This is the starting point of the project for SOEN341.

### CODE REVIEW

**Review by Alex-Hayeur — APPROVED** [review]

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,5 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (kwreen) [inline .idea/misc.xml:1]: These .idea files, are they user-specific?

**Comment by davidberard2** [comment]
This seems adapted to the needs, although we need to consider the possibility of using SQL to better meet our needs. The code seems to work well to show the connection between database and app.

**Comment by LHuss** [comment]
When using firebase, is the full database loaded into the application, or is only the data that is being referenced loaded?

**Comment by cxt714** [comment]
dependencies {
    compile fileTree(dir: 'libs', include: ['*.jar'])
    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
        exclude group: 'com.android.support', module: 'support-annotations'
    })
    compile 'com.android.support:appcompat-v7:26.+'
    compile 'com.android.support.constraint:constraint-layout:1.0.2'
    compile 'com.google.firebase:firebase-database:10.0.1'
    compile 'com.google.firebase:firebase-core:10.0.1'
    compile 'com.google.firebase:firebase-auth:10.0.1'
    compile 'com.android.support:design:26.+'
    testCompile 'junit:junit:4.12'
}


Reading the coding above,  I guess the data is referenced loaded.

**Pull request merged**